### ISSUE TITLE
Missing dependency 'react-query'

### ISSUE THREAD

**Issue by andinfarrel** [nexxeln/create-t3-app#26]
https://github.com/nexxeln/create-t3-app/blob/8c9f0f2f33fb555109728341e9c628790728aa45/src/installers/prisma.ts#L12:L25
tRPC requires `react-query` to be installed
```
await installPkgs({
    packageManager,
    projectDir,
    packages: [
      "@trpc/server",
      "@trpc/client",
      "@trpc/next",
      "@trpc/react",
      "react-query", // <-- here
      "superjson",
      "zod",
    ],
    devMode: false,
});
```

**Comment by nexxeln** [comment]
Thank you for spotting this, will be fixed soon.

**Comment by nexxeln** [comment]
Fixed in e0c9d4e
